Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -266,6 +456,19 @@ private LoginResult ConnectToArchipelago()
                 // Read slot data.
                 slot = success.Slot;
                 SlotOptions = SlotDataExtract.ExtractObjectFromSlotData<SlotOptions>(success.SlotData["options"]);
+                LogDebug($"Deathlink type: {SlotOptions.DeathLink}");
+
+                // Enable Deathlink
+                if(SlotOptions.DeathLink != DeathLinkType.None)
+                {
+                    deathLinkService = session.CreateDeathLinkServiceAndEnable();
+                    deathLinkService.OnDeathLinkReceived += OnDeathLinkReceived;
+                    session.UpdateConnectionOptions(new[] { "DeathLink" }, ItemsHandlingFlags.AllItems);
```

Comment:
The `CreateDeathLinkServiceAndEnable()` should automatically set the `DeathLink` tag. So you don't need to update connection options manually.

Reply:
I forgot to remove this when I discovered there was a built-in Deathlink Service.  Earlier I was constructing and listening for the packets by hand.